proxy-init/cmd/root.go

```diff
@@ -34,10 +34,10 @@ func NewRootCmd() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "proxy-init",
-		Short: "Adds a Kubernetes pod to the Conduit Service Mesh",
-		Long: `proxy-init Adds a Kubernetes pod to the Conduit Service Mesh.
+		Short: "proxy-init adds a Kubernetes pod to the Linkerd service mesh",
```

Comment:
just for my own information, why make short and long the same here?

Reply:
Good question -- this is for consistency with the `linkerd` CLI, which uses the same sentence for both Short and Long description, here:

https://github.com/linkerd/linkerd2/blob/master/cli/cmd/root.go#L32

Reply:
mostly for my own understanding: Is there actually anything Kubernetes-specific about this? My understanding is more that this should work within any linux namespace?

It might be nice to make this long description something less marketing and more technical: this tool configures the current Linux namespace such that all TCP traffic is proxied through the provided ports

Reply:
Honestly I'm not too familiar with proxy-init, but I think your understanding is correct. I've updated the long description to remove the linkerd to conduit.io, but I think I'm going to leave the wording the same as the short description for now. That's at least consistent with how descriptions are handled in the main CLI, and we can revisit as part of #1318.